src/game/default/g_types.h

```diff
@@ -515,6 +523,8 @@ typedef struct {
 
 		uint16_t weapon_no_ammo;
 		uint16_t weapon_switch;
+		
+		uint16_t countdown[11];
```

Comment:
That's exactly what `lengthof` does. It's defined as `(sizeof(x) / sizeof(x[0]))`.